README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcodecov%2Fexample-python?ref=badge_large)
 
-test
+test2
```

Comment:
The change from 'test' to 'test2' doesn't provide any meaningful information. Consider replacing this with actual content that describes the project, its features, or usage instructions.
```suggestion
# Example Usage

```python
import example_package

result = example_package.main_function()
print(result)
```
```